### DISCUSSION TITLE
What devices support the Announce feature?

### DISCUSSION THREAD

**Discussion by dw1562** [nimroddolev/chime_tts#142] [General]
I've been using the Google Assistant SDK Integration for a while purely so TTS announcements resumed any playing audio after the announcement.  However, Google seems to have killed this off for Workspace accounts.  Then I came across Chime TTS.  It seemed to cover all my needs and then some.   It works perfectly when I set my target devices as my Apple Homepod Minis.  However, it does not work properly if I set the targets to be Google Nest Hub/Mini devices.  It adjusts the volumes, plays the chime and the message, readjusts the volume but does not resume the audio that was playing.  

I see that the Announce feature only works on supported devices but I thought with the Google nest devices being so popular that they would be supported.  

Is there a list of supported devices?   I'd really like Chime to work with the Google devices if possible so I can set certain announcements in the rooms where I have them.

Thanks for a great integration!

BTW - I am running 1.1.0 beta 13

**Comment by nimroddolev** [comment]
Hi @dw1562, thanks for raising this issue and for the support 👍

The announce feature is long known to be an issue in Home Assistant. From the perspective of Chime TTS, I am limited by the features that each media_player platform supports and unfortunately some media_player platforms lack support for this feature likely due to to limitations from the manufacturers.

Some platforms (like Sonos) offer full announce support, some (like Apple TV) don't offer support for announce but they offer enough for me to be able to simulate it by using the reference to the currently playing media. However there are some platforms which I haven't found a workaround for, and I am guessing that the Google Assistant SDK Integration might be one of them, unfortunately.

I suggest you search the Home Assistant forum, as perhaps there are others who might have already posted on this issue. If you find a solution there please reply here so that others can make use of it, and perhaps I can integrate it into Chime TTS.
